### ISSUE TITLE
Email notifications should not include summaries from limited posts

### ISSUE THREAD

**Issue by 4ndyD** [diaspora/diaspora#4266]
At present, when someone else comments on a limited post that a user has made or commented on, while their email notifications are switched on, that user will receive an unencrypted email that includes a few sentences from the start of the latest comment, and using the first sentence of the original post in the email's subject line.

This presents a vulnerability to breach of privacy. Some people may still want that summary in their limited notifications, but only after providing D* with a public key to use so that they can receive it securely.

IMO this should be Murphy's-Law-safe, i.e. "if something can be done wrong, someone will eventually do it wrong", so ideally there should be no summary or subject for such notifications, merely a note that someone has responded, with a link to the relevant post. Otherwise, those who do need that privacy may be too worried about their contacts' unknown security measures to post some things.

There are plenty of examples of social networks and forum software that do not include the content of a private message in email notifications, and such a feature could even save a tiny fraction of server time & bandwidth by not having to fetch and write that data into emails.

**Comment by svbergerem** [comment]
When someone comments on a limited post the subject still includes the beginning of the limited post.

**Comment by svbergerem** [comment]
Fixed by #5843.

**Comment by 4ndyD** [comment]
When I get a private message from someone, an email notification of it still includes the subject of the PM in the subject line of the email. Does anyone else think this should be removed as well?

**Comment by svbergerem** [comment]
@4ndyD Thanks for reporting this. See #6850.

**Comment by 4ndyD** [comment]
@svbergerem Thanks, I wasn't sure if a new issue was justified.
